### ISSUE TITLE
`/reload` command fails with `NoSuchMethodError`

### ISSUE THREAD

**Issue by irisfofs** [fzzyhmstrs/fconfig#18]
Fzzy Config 0.5.3
Minecraft 1.20.1
Forge 47.2.8

Running `/reload` fails, and the server log contains the following error message:

```
java.lang.NoSuchMethodError: 'java.util.List net.minecraftforge.event.OnDatapackSyncEvent.getPlayers()'
        at TRANSFORMER/fzzy_config@0.5.3+1.20.1+forge/me.fzzyhmstrs.fzzy_config.networking.NetworkEvents.registerDataSync(NetworkEvents.kt:122)
```

It looks like `getPlayers()` was added in Forge 47.3. 

So this is kind of on me for using an out of date version of Forge, but if Fzzy Config is only compatible with a later version of Forge I would expect it to complain at startup.

**Comment by fzzyhmstrs** [comment]
Thanks for letting me know. I'll tweak the dependency to be 47.3 and up in 0.5.4

**Comment by fzzyhmstrs** [comment]
fixed in 0.5.4
